components/feature/autofill/src/main/java/mozilla/components/feature/autofill/ui/AbstractAutofillConfirmActivity.kt

```diff
@@ -49,7 +50,8 @@ abstract class AbstractAutofillConfirmActivity : FragmentActivity() {
 
         // While the user is asked to confirm, we already try to build the fill response asynchronously.
         dataset = lifecycleScope.async(Dispatchers.IO) {
-            fillHandler.handleConfirmation(structure, loginId)
+            val builder = fillHandler.handleConfirmation(structure?.toRawStructure(), loginId)
```

Comment:
If the `structure` is somehow `null` here, we can avoid invoking `lifecycleScope.async` entirely, right?